ui/src/app/cohort-search/attributes-page/attributes-page.component.ts

```diff
@@ -39,11 +39,12 @@ export class AttributesPageComponent implements OnDestroy, OnInit {
   subscription: Subscription;
   rangeAlert = false;
   loading: boolean;
+  selectedCode: any;
```

Comment:
This needs to have a value assigned to it in ngOnInit for PM and in radioChange function for measurements. Otherwise param id can get set to 'paramNaN' because it's undefined, which will also allow adding 2 selections for 'Any' in PM